src/inmanta/parser/plyInmantaParser.py

```diff
@@ -67,8 +67,10 @@
     ("left", "CMP_OP"),
     ("nonassoc", "NOT"),
     ("left", "IN"),
-    ("right", "MLS"),
-    ("right", "MLS_END"),
+    ("left", "RELATION_DEF", "TYPEDEF_INNER", "OPERAND_LIST", "EMPTY"),
+    ("left", "CID", "ID"),
+    ("left", "MLS"),
```

Comment:
```suggestion
    ("right", "MLS"),
```
We produce `mls` from end to start, which I believe means right associativity.